video/out/vo_gpu_next.c

```diff
@@ -2299,10 +2299,10 @@ AV_NOWARN_DEPRECATED(
     pars->color_map_params.contrast_smoothness = opts->tone_map.contrast_smoothness;
     pars->color_map_params.gamut_mapping = gamut_modes[opts->tone_map.gamut_mode];
 
+    pars->params.dither_params = NULL;
+    pars->params.error_diffusion = NULL;
+
     switch (opts->dither_algo) {
-    case DITHER_NONE:
```

Comment:
not sure if we have enum warnings enabled but it could make sense to keep the empty case

Reply:
We don't have them enabled. And I don't like empty cases. Unless we want to fix all cases like that I would prefer to just skip it here.